src/PointwiseFunctions/AnalyticSolutions/Poisson/Lorentzian.cpp

```diff
@@ -4,68 +4,59 @@
 #include "PointwiseFunctions/AnalyticSolutions/Poisson/Lorentzian.hpp"
 
 #include <array>
+#include <cstddef>
 
-#include "DataStructures/DataVector.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"
 #include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
-#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"
 #include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
 #include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
 #include "Utilities/MakeWithValue.hpp"
-// IWYU pragma: no_forward_declare Tensor
 
-namespace Poisson {
-namespace Solutions {
+/// \cond
+namespace Poisson::Solutions::detail {
 
-template <>
-tuples::TaggedTuple<Tags::Field> Lorentzian<3>::variables(
-    const tnsr::I<DataVector, 3>& x,
-    tmpl::list<Tags::Field> /*meta*/) noexcept {
-  return {Scalar<DataVector>(1. / sqrt(1. + get(dot_product(x, x))))};
+template <typename DataType, size_t Dim>
+void LorentzianVariables<DataType, Dim>::operator()(
```

Comment:
And here - same design change from `variables` specializations to `operator()`s in a detail class